server/plugins/connect-plugin-mysql/src/main/java/com/oceanbase/odc/plugin/connect/mysql/MySQLConnectionExtension.java

```diff
@@ -83,4 +84,12 @@ protected HostAddress parseJdbcUrl(String jdbcUrl) throws SQLException {
         return new HostAddress(url.getHost(), url.getPort());
     }
 
+    @Override
+    protected Map<String, String> appendDefaultJdbcUrlParameters(Map<String, String> jdbcUrlParams) {
+        if (!jdbcUrlParams.containsKey("tinyInt1isBit")) {
+            jdbcUrlParams.put("tinyInt1isBit", "false");
```

Comment:
Could you please explain what this parameter named 'tinyInt1isBit' means and any other outcomes after setting this one to false?

Reply:
msyql jdbc will automatically convert the data type of `tinyint(1)` to `Bit` type, causing ODC to display the data of tinyint(1) in the form of Bit data. But Navicat does not display tinyint(1) data as Bit type. Through research, it is found that this conversion can be avoided by setting `tinyInt1isBit=false` in jdbc parameters, so the `appendDefaultJdbcUrlParameters` method is added to `MySQLConnectionExtension` to set the default jdbc parameters.